### ISSUE TITLE
Selected option in the Select component doesn't have enough contrast

### ISSUE THREAD

**Issue by dobe0002** [instructure/instructure-ui#19]
When a user activates a Select drop-down, the selected option has white text on a dark gray background. This "out of the box" color combination does not have enough contrast to meet WCAG 2.0 AA success criteria 1.4.3.   The this sized text, there needs to be at least 4.5:1 ratio.  It currently appears to be around 3:1

Many institutions, including my own (University of Minnesota) use WCAG 2.0 AA as our accessibility requirement.

**Comment by arlodesign** [comment]
We offer a [high-contrast theme](https://instructure.design/#canvas-high-contrast) that meets this standard. This theme is available to all users of Canvas or consumers of InstructureUI. We are always evaluating our accessibility standards and will consider your feedback, but right now there are no plans to update this color. Thanks.
